src/streamlink/plugins/zdf_mediathek.py

```diff
@@ -85,11 +84,11 @@ def _extract_streams(self, response):
             for format_ in priority["formitaeten"]:
                 yield self._extract_from_format(format_)
 
-    def _parse_track(self, track, parser):
+    def _parse_track(self, track, parser, name):
         try:
             return parser(self.session, track["uri"])
         except IOError as err:
-            self.logger.error("Failed to extract {0} streams: {1}", name, err)
+            self.logger.debug("Failed to extract {0} streams: {1}", name, err)
```

Comment:
@back-to 
Could you at least limit to one line error print when it's geoblocked? :)